src/controllers/consumer.ts

```diff
@@ -253,6 +254,46 @@ export const downloadPublishedDataset = async (req: Request, res: Response, next
   }
 };
 
+export const getPostgresPivotTable = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+  /*
+  #swagger.hidden = true
```

Comment:
Try 

```
// #swagger.ignore = true
```

and re-run `npm run docs:generate`

https://swagger-autogen.github.io/docs/endpoints/ignoring-endpoint